.eslintrc.js

```diff
@@ -1,13 +1,21 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
 module.exports = {
     root: true,
+    extends: './packages/pwa-kit-dev/src/configs/eslint/eslint-config.js',
```

Comment:
Updating this root eslint config, so that the scripts folder and anything else outside the packages folder are linted too.